docs/howto/kubernetes/gardener/create-shoot-cluster.md

```diff
@@ -0,0 +1,113 @@
+# Create Kubernetes cluster using Cleura Cloud Control Panel 
+<!-- Config interpolation for the gui name didn't work for the title here. -->
+
+If you want to create a Kubernetes cluster, you can do it via {{config.extra.gui}} using Gardener.
+This tutorial shows you how to do that, and how to deploy a sample application on such cluster.
+
+## Prerequisites
+
+To access the cluster from your computer, you will need [kubectl](https://kubernetes.io/docs/tasks/tools/) installed on your machine.
```

Comment:
I'd suggest to use the anchor link here (that is, <https://kubernetes.io/docs/tasks/tools/#kubectl>).